src/dfind/dfind.c

```diff
@@ -539,6 +624,38 @@ int main (int argc, char** argv)
     	}
     }
 
+    /* create new mfu_file objects */
+    mfu_file_t* mfu_file = mfu_file_new();
+
+#ifdef DAOS_SUPPORT
+    rc = daos_init();
+
+    /* If DAOS "path" then path needs to be reset to path within DAOS container */ 
+    const char** argpaths = (const char**)(&argv[optind]);
+
+    struct duns_attr_t dattr  = {0}; 
```

Comment:
I see this struct is passed as a value to one function, but it's not currently used outside of that function.  Can that be moved to be a local within the function, or do you mean to have the function fill it in and return it?

Reply:
@adammoody fixed in #339, I just used a local variable in the function